config/hsc/processCcdWithFakesDriver.py

```diff
@@ -0,0 +1,10 @@
+"""
+hsc-specific overrides for processCcdWithFakesDriverTask
+"""
+import os.path
+
+
+config.doMakeSourceTable = True
+#config.processCcdWithFakes.load(os.path.join(os.path.dirname(__file__), "processCcdWithFakes.py"))
```

Comment:
Remove commented lines.

Reply:
Good catch, this actually was waiting on another ticket being merged so that it could work. Looking into this made me realise that I was missing a processCcdWithFakes.py in obs_subaru/config/hsc/.